ibutex.py

```diff
@@ -64,6 +95,8 @@
 if not args.quick:
 	subprocess.call(fullbib)
 	subprocess.call(fullcmd)
+	if args.sagemath:
+		subprocess.call(fullsage)
```

Comment:
Is there any particular reason why a full latex command needs to be run between bibtex and sagetex? Otherwise it would make more sense to call sage right after bibtex, doing all the outside processing in one step.

Reply:
yes that is required … the reason being that the sage script that will get run is generated by the from the latex source ;)

Reply:
but one latex-run should suffice, shouldn't it? [we have one here](https://github.com/cherti/ibutex/blob/main/ibutex.py#L59), if that run would prepare everything we need already, we could then apply bibtex, sage and all other external processors that we intend to use, en bloc and afterwards run latex another time (twice without changing anything just in case because it's latex). Do you think that would work as well?